mappings/net/minecraft/class_193.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_193
```

Comment:
why are you naming stuff in classes without a name conc

Reply:
I didn't name that one :concern:

Reply:
what if there are obvious fields like `world` or `player` in an unnamed class? I think name those fields. Or if there's an obvious `instance` and `getInstance()` pair, I'd say name those as well. Mappings in unnamed classes were abundant in earlier Textile days. I would argue mappings in this way are fine.

Reply:
See [`class_67`](https://github.com/TextileMC/yarrn/blob/master/mappings/net/minecraft/class_67.mapping), [`class_100`](https://github.com/TextileMC/yarrn/blob/master/mappings/net/minecraft/class_100.mapping), [`class_157`](https://github.com/TextileMC/yarrn/blob/master/mappings/net/minecraft/class_157.mapping), [`class_195`](https://github.com/TextileMC/yarrn/blob/master/mappings/net/minecraft/class_195.mapping), [`class_222`](https://github.com/TextileMC/yarrn/blob/master/mappings/net/minecraft/class_222.mapping), and [`class_279`](https://github.com/TextileMC/yarrn/blob/master/mappings/net/minecraft/class_279.mapping)

Reply:
However, [`class_67`](https://github.com/TextileMC/yarrn/blob/master/mappings/net/minecraft/class_67.mapping) could be mapped to `StepSound` or similar (see [`Block`](https://github.com/TextileMC/yarrn/blob/480f6e39aba376bb0694c555f1409c13910b5689/mappings/net/minecraft/block/Block.mapping#L15-L20) mappings).